webClient/src/app/app.component.html

```diff
@@ -36,6 +37,14 @@
 
         <input type="radio" [(ngModel)]="actionType" name="actionType" value="Message">
         <label for="actionMessage" i18n="send a message to another app@@message">Message</label>
+
+
+        <input type="radio" [(ngModel)]="actionType" name="actionType" value="Maximize">
+        <label for="actionMessage" i18n="send a message to another app@@message">Maximize</label>
```

Comment:
These will not internationalize correctly because the i18n string (i18n is shorthand for "internationalization") will substitute in the string for "Message". You could come up with another such as:
`i18n="maximize an app instance@@maximize"`